src/test/jtx/impl/Env.cpp

```diff
@@ -562,12 +563,37 @@ Env::ust(JTx const& jt)
 
 Json::Value
 Env::do_rpc(
+    rpcCallback cb,
     unsigned apiVersion,
     std::vector<std::string> const& args,
     std::unordered_map<std::string, std::string> const& headers)
 {
-    return rpcClient(args, app().config(), app().logs(), apiVersion, headers)
-        .second;
+    // We shouldn't need to retry, but it fixes the test on macOS for
+    // the moment.
+    if (!cb)
+    {
+        static auto const defaultCallback = [](Json::Value const& jr) {
+            auto const parsedResult = parseResult(jr, false);
+            return parsedResult.rpcCode != rpcINTERNAL;
+        };
+        cb = defaultCallback;
```

Comment:
Can we move this out of the function, into the definition of `useDefaultCallback` (either inline or extern), and rename `useDefaultCallback` to something like `rejectInternalError`? If you prefer to frame the predicate names as `accept{Condition}` instead of `reject{Condition}`, then I'd be fine with that too.

Reply:
Yes. Fixed.